src/Common/MetricHelper.cs

```diff
@@ -303,6 +379,31 @@ private void PopulatePropertiesFromQos(AzurePSQoSEvent qos, IDictionary<string,
             }
         }
 
+        private static string[] exceptionTrackAcceptModuleList = { "Az.Accounts", "Az.Compute", "Az.AKS", "Az.ContainerRegistry" };
+        private static string[] exceptionTrackAcceptCmdletList = { "Get-AzKeyVaultSecret", "Get-AzKeyVaultCert" };
+
+        private static string ConvertFrameToString(StackFrame frame)
+        {
+            string[] fullNameParts = frame?.GetMethod()?.DeclaringType?.FullName?.Split('.');
```

Comment:
It seems that DeclaringType.FullName doesn't contain method name, so method name may be missed?

Reply:
Great.